### PR TITLE
Landing page setup

### PR DESCRIPTION
**Pull request by dev-enforced**
In this pull request user would see the user onboarding page which will take the name of the user (after pressing enter key) and then will proceed to the dashboard page.

### CODE REVIEW

File: src/pages/DashboardPage/DashboardPage.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
+const DashboardPage = () => {
+  return (<></>);
```
Comment (Kothalkarpriya) [inline src/pages/DashboardPage/DashboardPage.tsx:3]: Page is empty

File: src/pages/UserOnboardingPage/UserOnboardingPage.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    ) {
+      const keyEventValueCapitalised = capitaliseUsername(
+        keyEvent.target.value
+      );
+      setUsername(keyEventValueCapitalised);
+      localStorage.setItem("individual_name", keyEventValueCapitalised);
+    }
+  };
+  return (
+    <>
```
Comment (Kothalkarpriya) [inline src/pages/UserOnboardingPage/UserOnboardingPage.tsx:26]: It is not needed here as the code is already inside div block

File: src/types/contextTypes/BrowserContext.types.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+export type BrowserContextType = {
+  username: string;
```
Comment (Kothalkarpriya) [inline src/types/contextTypes/BrowserContext.types.ts:3]: good that you are using typescript

**Review by Kothalkarpriya — APPROVED** [review]
Good code structure and easily understandable. You can merge the changes after reviewing the suggestions

File: src/pages/DashboardPage/DashboardPage.tsx (hunk 1 above)
Comment (dev-enforced) [reply src/pages/DashboardPage/DashboardPage.tsx:3]: Yes that will be developed in the future PR

**Review by sushil-mod — APPROVED** [review]
Good start with typescript . Found some new things to learn from your code. Good code

**Pull request merged**